penlight-dev-1.rockspec

```diff
@@ -1,18 +1,16 @@
-local package_name = "penlight"
-local package_version = "dev"
-local rockspec_revision = "1"
-local github_account_name = "lunarmodules"
-local github_repo_name = package_name
-local git_checkout = package_version == "dev" and "master" or package_version
-
+package = "penlight"
+local rock_version = "dev"
+local rock_release = "1"
+local namespace = "lunarmodules"
+local repository = package
 
 rockspec_format = "3.0"
-package = package_name
-version = package_version .. "-" .. rockspec_revision
+version = ("%s-%s"):format(rock_version, rock_release)
```

Comment:
I think the concatenations are easier to read for non-Lua folks than the format-placeholder approach.